.ci/windows-steps.yaml

```diff
@@ -12,8 +12,8 @@ steps:
     nuget install boost -o $(Agent.ToolsDirectory) -Version 1.60.0
     nuget install boost_random-vc140 -o $(Agent.ToolsDirectory) -Version 1.60.0
     nuget install boost_math_c99-vc140 -o $(Agent.ToolsDirectory) -Version 1.60.0
-    nuget install OpenBLAS -o $(Agent.ToolsDirectory)
     nuget install unofficial-flayan-cereal -o $(Agent.ToolsDirectory)
+    nuget install ensmallen:x64-windows -o $(Agent.ToolsDirectory)
```

Comment:
Maybe the `:x64-windows` isn't needed here?  I am not sure.

Reply:
Let me know if you want me to try and debug this one.  I am sure the build will work right; we just have to convince Windows to download ensmallen correctly, and then MSVC to look in the correct path. :)

Reply:
Agreed, ensmallen is not in `nuget` I have opened an issue in ensmallen, [here](https://github.com/mlpack/ensmallen/issues/311), but no one volunteered, I have been looking into it last night, and it seems to me (not sure yet) that it might be possible to use Linux to create a nuget package.
I will give a try tomorrow if I have a spare time.

Reply:
It is available in vcpkg though (https://github.com/microsoft/vcpkg/issues/5015), so maybe we can manage to use that instead?  I can't remember if we already tried it or what the outcome there was.